src/_locales/it/messages.cson

```diff
@@ -182,3 +182,159 @@ simultaneousTasks:
 	message: 'Attività di download contemporanee:'
 unlimited:
 	message: 'Senza limiti'
+editFileSize:
+	message: 'Dimensione:'
+openInSidebar:
+	message: 'Apri nella sidebar'
+e_saveFileError:
+	message: 'Salvataggio fallito'
+e_threadStopped:
+	message: 'Interruzione imprevista'
+legacyFilenameEncoding:
+	message: 'Codifica nomi per siti legacy:'
+address:
+	message: 'Indirizzo'
+enterAddress:
+	message: 'Inserisci qui la lista degli indirizzi di download'
+selectAddressFromList:
+	message: 'Seleziona l\'indirizzo di download dalla lista.'
+invalidAddress:
+	message: 'L\'indirizzo di download non è valido.'
+copyAddress:
+	message: 'Copia indirizzo'
+filenamePlaceholder:
+	message: 'Inserisci il nome del file, della sottocartella o del template'
+convert:
+	message: 'Converti'
+link:
+	message: 'Link'
+media:
+	message: 'Media'
+openFile:
+	message: 'Apri file'
+openContainingFolder:
+	message: 'Apri cartella di destinazione'
+copyLink:
+	message: 'Copia link di download'
+openReferrer:
+	message: 'Apri pagina sorgente'
+remove:
+	message: 'Rimuovi'
+deleteFile:
+	message: 'Elimina file'
+actionTitleWithShift:
+	message: '$1\nShift: $2'
+text:
+	message: 'Testo'
+pageInaccessibleToExtensions:
+	message: 'Questa pagina non è accessibile dall'estensione.'
```

Comment:
Here is the missing `\'`.

It can be found by looking at the changed colors below.

Reply:
Thanks will update.